client/lib/plans/features-list.js

```diff
@@ -278,12 +277,26 @@ export const FEATURES_LIST = {
 		},
 	},
 
-	[ FEATURE_ADVANCED_CUSTOMIZATION ]: {
-		getSlug: () => FEATURE_ADVANCED_CUSTOMIZATION,
-		getTitle: () => i18n.translate( 'Advanced customization' ),
+	[ WPCOM_FEATURES_GLOBAL_STYLES ]: {
+		getSlug: () => WPCOM_FEATURES_GLOBAL_STYLES,
+		getTitle: () =>
+			i18n.translate( 'Advanced design customization', {
+				components: {
+					strong: <strong />,
+				},
+			} ),
```

Comment:
See my note in the summary:

> I intentionally keep using `WPCOM_FEATURES_GLOBAL_STYLES` for the "Advanced design customization" feature, since we're just giving a more use-friendly name to the Global Styles feature, but internally we're using `WPCOM_FEATURES_GLOBAL_STYLES` all over the place, so I preferred to be consistent here.

Reply:
> internally we're using a global styles terminology all over the place, so I preferred to be consistent here.

I realize this is not consistent with how we're dealing with the CSS customization feature though, but that's because that feature was introduced before the Limited Atomic pod added the new WPCOM features API (see p58i-cDG-p2).

In a follow-up PR, we should sync the same feature code across WPCOM (`class-wpcom-features.php`) and Calypso (`features.ts`), and rename the constant in Calypso to better reflect the synced feature code.

Reply:
Nah, on a second thought, I think it's better to leave the whole WPCOM/Calypso features sync for later, and keep using a constant name that doesn't match the WPCOM feature.